app/src/test/java/org/greenstand/android/TreeTracker/utils/FakeFileGenerator.kt

```diff
@@ -44,4 +48,144 @@ val emptyUser = User(
     isPowerUser = true,
     photoPath = "some random text",
     unreadMessagesAvailable = false
-)
\ No newline at end of file
+)
+
+val fakePlanterInfo = PlanterInfoEntity(

```

Comment:
Can we wrap all this fake data into an object, that way we don't have to remember any names, we just type FakeData.fakePlanterData etc... the auto complete will help a lot in finding data.

Reply:
Done